src/mozmlops/templates/README.md

```diff
@@ -142,18 +142,27 @@ Deploying your Ray Serve app to production requires changes in 3 different repos
 1. Install requirements
 
     ```sh
-    pip install -r requirements-rayserve.txt
+    pip install -r requirements.rayserve.txt
     ```
-2. Run Ray Serve app locally using [serve run](https://docs.ray.io/en/latest/serve/api/index.html#serve-run) command
+   
+1. Somewhere inside the environment where you've installed your requirements, run the following two lines of Python:
+    
+   ```python
+    import ray
+    ray.init()
+    ```
+   You only have to do this once per environment. It's commented out at the top of template_ray_serve.py, so you can uncomment it there the first time you do the next step in this list if you like.
+
+1. Run Ray Serve app locally using [serve run](https://docs.ray.io/en/latest/serve/api/index.html#serve-run) command
 
     ```sh
     serve run template_ray_serve:translator_app --route-prefix "/translate"
     ```
-3. In a different terminal, call the locally running service endpoint and check if it returns the expected response
+1. In a different terminal, call the locally running service endpoint and check if it returns the expected response
     ```sh
     curl -i -d '{"text": "Hello world!"}' -X POST "http://127.0.0.1:8000/translate/" -H "Content-Type: application/json"
     ```
-4. Stop the Ray Serve app (kill the `serve run` process) after you are done with testing
+1. Stop the Ray Serve app (kill the `serve run` process) after you are done with testing
```

Comment:
I changed all the numbers to 1s because in Markdown syntax, this is how you say the equivalent of <li> in an <ol> in HTML. This way Markdown handles numbering things correctly for you and you don't have to go back and renumber stuff when you add a step in the middle.